tests/experimental/datasources/test_config.py

```diff
@@ -23,7 +23,7 @@
 )
 
 try:
-    from devtools import debug as pp
+    from devtools import debug as pp  # noqa: disable=E0602
```

Comment:
I wonder why this error disable is necessary now when it wasn't before. Could you declare `pp` before this try block to remove this `noqa`? That is `pp: Callable[...]` and set `...` to whatever it should be.

Reply:
@billdirks The `noqa` is needed because to disable `PyCharm` warning (which says "devtools in try block should also be in the except block"); however, I also declared the `Callable` based on your suggestion.  Thanks.

Reply:
It shouldn't be needed. This is a PyCharm warning, not a linting or type-checker warning.
@alexsherstinsky I think you should revert this change.

Reply:
@Kilo59 Commited -- thanks for pairing and the devtools docs!